test/api/api_test.go

```diff
@@ -205,13 +205,13 @@ func TestUserTierCache(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	var ul database.TierLimits
+	var ul database.UserLimitsResponse
 	err = json.Unmarshal(b, &ul)
 	if err != nil {
 		t.Fatal(err)
 	}
-	if ul.TierName != database.UserLimits[database.TierFree].TierName {
```

Comment:
We should still check for the id and the name for backwards compatibility.

Reply:
OK.